devops/ansible/examples/girder-dev-environment/site.yml

```diff
@@ -70,3 +70,11 @@
       become: yes
       become_user: root
       when: ansible_os_family == "RedHat"
+
+    - name: Activate virtualenv on login
+      lineinfile:
+        dest: ~/.profile
+        line: 'source {{girder_virtualenv}}/bin/activate'
+        insertafter: 'EOF'
+        regexp: 'source {{girder_virtualenv}}/bin/activate'
```

Comment:
This `regexp` parameter is not necessary, as [an omitted `regexp` will cause the module to match the literal `line`](https://github.com/ansible/ansible/blob/v2.3.2.0-1/lib/ansible/modules/files/lineinfile.py#L385). Futhermore, explicitly passing the `line` value as `regexp` will cause the value to be interpreted as a regex, which we definitely are not expecting. In this case, it will just match the `.` as a wildcard, but any other regex-special characters in the path would cause it to break. Furthermore, explicitly passing this argument conveys the semantics that both `line` and `regexp` are required, even if they are the same value, which may entice other developers to copy-paste this pattern and encounter into the special-regex-character issue.